data-processing/scripts/process_pdf.py

```diff
@@ -64,10 +67,10 @@ def union(self, bbox1: BoundingBoxModel, bbox2: BoundingBoxModel):
     @staticmethod
     def should_combine(bbox1: BoundingBoxModel, bbox2: BoundingBoxModel):
         return bbox1.page == bbox2.page and (
-            abs(bbox1.top - bbox2.top) < 4 # Same y-coordinate
+                abs(bbox1.top - bbox2.top) < 4 # Same y-coordinate
```

Comment:
Plus some spacing changes?

Reply:
ooops, that happened somehow from copy/paste.